crypto/encode_decode/decoder_lib.c

```diff
@@ -632,12 +734,20 @@ static int decoder_process(const OSSL_PARAM params[], void *arg)
 
         OSSL_TRACE_BEGIN(DECODER) {
             BIO_printf(trc_out,
-                       "(ctx %p) Running decoder instance %p => %d\n",
-                       (void *)new_data.ctx, (void *)new_decoder_inst, ok);
+                       "(ctx %p) %s [%u] Running decoder instance %p => %d\n",
+                       (void *)new_data.ctx, LEVEL, (unsigned int)i,
+                       (void *)new_decoder_inst, ok);
         } OSSL_TRACE_END(DECODER);
 
         if (ok)
             break;
+
+        /*
+         * These are errors are assumed to come from
```

Comment:
These errors are?

Reply:
... apparent in the code following the comment :wink:

Reply:
I was intending to comment on the grammar..shouldn't this be "These errors are assumed..."

Reply:
Oh!  My eyes failed me.  Ok, will fix